### ISSUE TITLE
Requirement management

### ISSUE THREAD

**Issue by caufieldjh** [Knowledge-Graph-Hub/NEAT#32]
Running NEAT after a fresh install yields a ModuleNotFoundError for `cpuinfo` due to `ensmallen` importing it.
This can be avoided with a `pip install ensmallen` - or even better, `pip install grape` as this should cover all requirements.

There may also be issues with importing Tensorflow if it is not already installed, as ensmallen/embiggen do not explicitly require it.

**Comment by caufieldjh** [comment]
Related - `transformers` requires `pyyaml` >=5.1 but this may not be present as NEAT does not specify a version

**Comment by caufieldjh** [comment]
This was the setup order that worked for me:
```
python setup.py install
pip install grape
pip install transformers
pip install urllib3==1.25.11
```

**Comment by justaddcoffee** [comment]
closed by #34
